docs/explanation/upgrade.rst

```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
 *all-in-one* method for upgrades. because its charm is designed to maintain service
 availability throughout the upgrade process.
 
+
+SRULP support
```

Comment:
```suggestion
SLURP support
```

The acronym is for "Skip Level Upgrade Release Process", ref. https://docs.openstack.org/project-team-guide/release-cadence-adjustment.html

Reply:
Thanks for verify